util/dvsim/utils.py

```diff
@@ -234,10 +234,10 @@ def color_cell(cell, cclass, indicator="%"):
     na_list_patterns = '|'.join(na_list)
 
     # List of floating point patterns: '0', '0.0' & '.0'
-    fp_patterns = "\d+|\d+\.\d+|\.\d+"
+    fp_patterns = "[\+\-]?\d+\.?\d*"
 
     patterns = fp_patterns + '|' + na_list_patterns
-    indicators = "%|E|W"
+    indicators = "%|%%|E|W|T|P"
```

Comment:
yeah I just hacked this in to get it going at this point. I'll clean this up.

I think it would still be great if we had something like a `%` symbol in there for those messages without markup, since that makes the generated .md (shown on the console) more readable.

Also, considering that we explicitly _add_ an identifier in cases we want special markup, it might make sense to encode percentages that way as well. I.e., `%` would then just get rendered as `%`. But if you use something like `%m` it would switch to the marked-up version.